libs/tup-components/src/news/UserNews.tsx

```diff
@@ -24,38 +33,44 @@ const ViewAllUpdates = () => (
 
 const UserNews: React.FC = () => {
   const { data, isLoading } = useUserNews();
+  const maxItems = 5;
+
   if (isLoading) return <LoadingSpinner />;
   return (
     <SectionTableWrapper
       header="User Updates"
       headerActions={<ViewAllUpdates />}
       contentShouldScroll
     >
-      <ul className={styles['news-list']}>
+      <div className={styles['news-list']}>
```

Comment:
A list makes sense, but each is also an `<article>`[^1], and `ul` → `li` → `article` is extra nesting that requires undoing browser or future list styles.

[^1]: The `<article>` tag is used for news on CMS in User News plugin and News app. I aim for consistency by doing the same here.